toolchains/linux.cmake

```diff
@@ -34,6 +34,14 @@ set(CORE_COMPILE_DEFS PLATFORM_LINUX)
 # load core library
 add_subdirectory(${PROJECT_SOURCE_DIR}/core)
 
+if("${MAPZEN_API_KEY}" STREQUAL "")
```

Comment:
It would make send to add this as a CMake routine to reduce deduplication, it's used on almost every single platform now.